demo/Vagrantfile

```diff
@@ -22,15 +24,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.synced_folder ".", "/vagrant", disabled: true
 
-  config.vm.provision "shell", privileged: false, inline: <<SCRIPT
-    ## Uncomment these lines to update flynn before starting
-    # sudo apt-get update -qq
-    # sudo apt-get install -y flynn-host
-    # sudo flynn-host download /etc/flynn/version.json
-
-    sudo start flynn-host
+  config.vm.provider "virtualbox" do |v|
+    v.gui = true
```

Comment:
Glad you asked. ;-)

I've worked with a lot of teams on-boarding them onto Vagrant. One of the really annoying things about Vagrant to someone who hasn't used it much (like a developer moving from Heroku to Flynn) is how it somewhat obfuscates the fact that `vagrant up` actually started a VM that is hogging a lot of resources. Or more precise, the fact that the dev has to run a `vagrant halt` to stop the VM from hogging resources. This is obvious to anyone who has Vagrant experience, but surprisingly easy to forget for non-Vagrant users.

Adding the gui flag causes VirtualBox and VMWare to show the normal KVM'ish interface and serves as a visual reminder that the VM is running.

Basically, I'm in favor of as little magic as possible when people are demoing things.